certification/internal/shell/scorecard_olm_suite.go

```diff
@@ -0,0 +1,134 @@
+package shell
+
+import (
+    "os/exec"
+    "os"
+    "fmt"
+    "io/ioutil"
+
+    "github.com/komish/preflight/certification"
+    "github.com/sirupsen/logrus"
+)
+
+type ScorecardOlmSuiteTest struct{}
+
+func (p *ScorecardOlmSuiteTest) Validate(bundle string, logger *logrus.Logger) (bool, error) {
+    
+    //Curentdir
+    homedir, err := os.Getwd()
```

Comment:
Check for the error before you use homedir.

Reply:
Done.